pelican/server.py

```diff
@@ -43,6 +43,21 @@ def parse_arguments():
 class ComplexHTTPRequestHandler(server.SimpleHTTPRequestHandler):
     SUFFIXES = ['.html', '/index.html', '/', '']
 
+    extensions_map = _encodings_map_default = {
+        # included in Python default implementation
+        '.gz': 'application/gzip',
+        '.Z': 'application/octet-stream',
+        '.bz2': 'application/x-bzip2',
+        '.xz': 'application/x-xz',
+
+        # web fonts
+        ".oft": "font/oft",
+        ".sfnt": "font/sfnt",
+        ".ttf": "font/ttf",
+        ".woff": "font/woff",
+        ".woff2": "font/woff2",
+    }
+
```

Comment:
I think it will be more future-proof if we extend over the original map instead of overriding it:

```python
    extensions_map = {
        **server.SimpleHTTPRequestHandler.extensions_map,
        **{
            # web fonts
            ".oft": "font/oft",
            ".sfnt": "font/sfnt",
            ".ttf": "font/ttf",
            ".woff": "font/woff",
            ".woff2": "font/woff2",
        }
    }
```

I looked at the CPython code and as far as I can see `_encodings_map_default`[*] serves no purpose at all. Seems like `extensions_map` is expected to be modified and that just keeps initial values as backup if it was needed afterwards. And looks like it wasn't needed. So I guess we should leave that one alone.

[*]: Also, looks like it was a typo and should be `_extensions_map_default` instead in the CPython code (?)